server/battleships/domain/src/main/java/eu/optas/domain/Board.java

```diff
@@ -0,0 +1,11 @@
+package eu.optas.domain;
+
+public class Board {
+    private final int[][] grid;
```

Comment:
Mes BE naudojam visur Lists, tad siūlyčiau naudoti juos ir čia.

Reply:
nžn ar verta čia taip complicatint :D List<List<Integer>> būtų